lib/krikri/search_results_helper_behavior.rb

```diff
@@ -42,5 +42,51 @@ def render_bookmarks_control?
       false
     end
 
+    # Render enriched record for view
+    # @param [Krikri::SearchIndexDocument]
+    # @return [String]
+    def render_enriched_record(document)
+      agg = document.aggregation
+      return error_msg unless agg.present?
+      JSON.pretty_generate(agg.to_jsonld)
+    end
+
+    # Render original record for view
+    # @param [Krikri::SearchIndexDocument]
+    # @return [String]
+    def render_original_record(document)
+      agg = document.aggregation
+      return error_msg unless agg.present?
+
+      begin
+        original_record = agg.original_record
+      rescue StandardError => e
+        logger.error e.message
+        return error_msg
+      end
+
+      prettify_string(original_record.to_s)
+    end
+
+    private
+
+    def prettify_string(string)
```

Comment:
Is that a "we want it to" or "it will"?  The implementation of `OriginalRecord#content_type` defaults to `text/xml`.

https://github.com/dpla/KriKri/blob/develop/app/models/krikri/original_record.rb#L107